custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json

```diff
@@ -2415,6 +2622,201 @@
         "case_id_path": [
           "child_health_case_id"
         ]
+      },
+      "stunting_last_recorded": {
+        "type": "switch",
+        "switch_on": {
+          "type": "nested",
+          "argument_expression": {
+            "aggregation_fn": "last_item",
+            "type": "reduce_items",
+            "items_expression": {
+              "type": "icds_get_case_forms_by_date",
+              "xmlns": [
+                "http://openrosa.org/formdesigner/7a55754119359466b1951d7251068bd4f45e73c3"

```

Comment:
How many times does this form get submitted per case? If it's a lot, this is probably going to be really expensive